.changeset/chilled-zoos-rush.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-api-docs': minor
```

Comment:
As this is a breaking change, I would use minor here? We removed some components that others might use. The stability is `0` so breaking changes are fine.

Reply:
Yep, this is correct.